internal/local/app_test.go

```diff
@@ -673,87 +698,39 @@ exports = function({ query }) {
-		"app_default_domain": "full-tkdcx.stitch-statichosting-dev.baas-dev.10gen.cc",
-	},
-	CustomUserDataConfig: map[string]interface{}{
-		"enabled":            true,
-		"mongo_service_name": "mongodb-atlas",
-		"database_name":      "test",
-		"collection_name":    "coll3",
-		"user_id_field":      "xref",
-	},
-	Sync: map[string]interface{}{"development_mode_enabled": false},
-}}
-
-var appData20200603Local = AppDataV1{AppStructureV1{
-	ConfigVersion:   realm.AppConfigVersion20200603,
-	Name:            "20200603-local",
-	Location:        realm.LocationVirginia,
-	DeploymentModel: realm.DeploymentModelGlobal,
-	Security:        map[string]interface{}{"allowed_request_origins": []interface{}{"http://localhost:8080"}},
-	Hosting: map[string]interface{}{
-		"enabled":            true,
-		"app_default_domain": "full-tkdcx.stitch-statichosting-dev.baas-dev.10gen.cc",
-	},
-	CustomUserDataConfig: map[string]interface{}{
-		"enabled":            true,
-		"mongo_service_name": "mongodb-atlas",
-		"database_name":      "test",
-		"collection_name":    "coll3",
-		"user_id_field":      "xref",
-	},
-	Sync: map[string]interface{}{"development_mode_enabled": false},
-}}
-
-var appData20200603Remote = AppDataV1{AppStructureV1{
-	ConfigVersion:   realm.AppConfigVersion20200603,
-	ID:              "20200603-remote-abcde",
-	Name:            "20200603-remote",
-	Location:        realm.LocationVirginia,
-	DeploymentModel: realm.DeploymentModelGlobal,
-	Security:        map[string]interface{}{"allowed_request_origins": []interface{}{"http://localhost:8080"}},
-	Hosting: map[string]interface{}{
-		"enabled":            true,
-		"app_default_domain": "full-tkdcx.stitch-statichosting-dev.baas-dev.10gen.cc",
-	},
-	CustomUserDataConfig: map[string]interface{}{
-		"enabled":            true,
-		"mongo_service_name": "mongodb-atlas",
-		"database_name":      "test",
-		"collection_name":    "coll3",
-		"user_id_field":      "xref",
-	},
-	Sync: map[string]interface{}{"development_mode_enabled": false},
-}}
+func createAppDataV1(version realm.AppConfigVersion, name string) AppDataV1 {
+	app := AppDataV1{AppStructureV1{
+		ConfigVersion:   version,
+		Name:            version.String() + "-" + name,
+		Location:        realm.LocationVirginia,
+		DeploymentModel: realm.DeploymentModelGlobal,
+		Security:        map[string]interface{}{"allowed_request_origins": []interface{}{"http://localhost:8080"}},
+		Hosting: map[string]interface{}{
+			"enabled":            true,
+			"app_default_domain": "full-tkdcx.stitch-statichosting-dev.baas-dev.10gen.cc",
+		},
+		CustomUserDataConfig: map[string]interface{}{
+			"enabled":            true,
+			"mongo_service_name": "mongodb-atlas",
+			"database_name":      "test",
+			"collection_name":    "coll3",
+			"user_id_field":      "xref",
+		},
+		Sync: map[string]interface{}{"development_mode_enabled": false},
+	}}
+
+	switch name {
+	case "remote":
+		app.AppStructureV1.ID = version.String() + "-" + name + "-" + "abcde"
```

Comment:
[meganit] this is probably a moot point based on my previous suggestion, this could just be:

`app.AppStructureV1.ID = app.AppStructureV1.Name + "-" + "abcde"`